buildpacks/python/python/build.go

```diff
@@ -67,10 +67,10 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return result, nil
 	}
 
-	functionClass, isFuncDefDefault := cr.Resolve("BP_FUNCTION")
+	functionClass, isOverride := cr.Resolve("BP_FUNCTION")
```

Comment:
Changes here and below are for consistency with java changes. Kept them minimal.